seed/models/data_quality.py

```diff
@@ -702,76 +735,64 @@ def _check(self, rules, row):
                 if (rule.table_name, rule.field) not in self.column_lookup:
```

Comment:
After doing some manual testing of this logic, here's what I'm seeing when comparing the old setup and this new setup:

**I think the following are working as expected:**

- In old and new setup, when "Address Line 2" only has `contain` Rules, mapping that column but values are null;  this does not trigger Rules. ![before_after - contains THEN mapped - GOOD](https://user-images.githubusercontent.com/30608004/80938920-1714b580-8d98-11ea-8887-941c71209848.png)

- In old setup, when "Conditioned Floor Area" has `not null` checked for the two `min` and `max` Rules, not mapping triggers both Rules. In new setup, when "Conditioned Floor Area" has three separate `not null, `min`, and `max` Rules, not mapping that column triggers only the `not null` Rule. ![before_after - minmax not_null THEN not mapped - GOOD](https://user-images.githubusercontent.com/30608004/80938923-1845e280-8d98-11ea-9d3b-a5f119ed1603.png)

- In old and new setup, when "Address Line 2" only has `contain` Rules, not mapping that column does not trigger Rules. ![before_after - contains THEN not mapped - GOOD](https://user-images.githubusercontent.com/30608004/80938925-18de7900-8d98-11ea-91c3-fd6e0c7554e9.png)

**I'm wondering if the following two cases are not what we'd expect:**

- In old setup, when "Conditioned Floor" area has only `min` and `max` Rules, mapping that column but values are null, this **does not trigger** Rules. In new setup, when "Conditioned Floor" area has only min and max Rules, mapping that column but values are null - this **does trigger** Rules. ![before_after - minmax THEN mapped to empty - BAD?](https://user-images.githubusercontent.com/30608004/80938917-14b25b80-8d98-11ea-8b62-1c5c33f319ce.png)

- In old setup, when "Conditioned Floor Area" has only `min` and `max` Rules, not mapping that column **does not trigger** Rules. In new set up, when "Conditioned Floor Area" has only `min` and `max` Rules, not mapping that column **does trigger** Rules. ![before_after - minmax THEN not mapped - BAD?](https://user-images.githubusercontent.com/30608004/80938922-1845e280-8d98-11ea-9d88-2c394f37b848.png)